joinpoint: revised (#47)
bcgov/opho-cdr-shiny: docs/report/capstone-report.Rmd

## Joinpoint Regression

When the response is dependent on the explanatory variable in a
piece-wise linear manner, such as when multiple linear pieces are

> connected at unknown points, segmented are used as regression models.

Reviewer: ```suggestion
connected at unknown points, segmented regression models are used.
```

> They also termed as or broken-line models. Here the response variable

Reviewer: ```suggestion
They also termed as broken-line models. Here the response variable
```

> are different standardized chronic disease rates, and time in years

Reviewer: ```suggestion
are different age-standardized rates, and time in years
```

> (2001 to 2020) being the explanatory variable. This type of model is

Reviewer: ```suggestion
(2001 to 2020) is the explanatory variable. This type of model is
```

> particularly applied in the filed of epidemiology, occupational

Reviewer: ```suggestion
particularly applied in the fields of epidemiology, occupational
```

> medicine, toxicology, and ecology, where often, the goal is to to

Reviewer: ```suggestion
medicine, toxicology, and ecology, where often, the goal is to
```
